evmjit/libevmjit/Compiler.cpp

```diff
@@ -109,12 +109,14 @@ void Compiler::resolveJumps()
 	// Iterate through all EVM instructions blocks (skip first one and last 4 - special blocks).
 	for (auto it = std::next(m_mainFunc->begin()), end = std::prev(m_mainFunc->end(), 4); it != end; ++it)
 	{
-		auto nextBlock = it->getNextNode(); // If the last code block, that will be "stop" block.
+		auto nextBlockIter = it;
+		++nextBlockIter; // If the last code block, that will be "stop" block.
+		
 		auto term = it->getTerminator();
 		llvm::BranchInst* jump = nullptr;
 
 		if (!term) // Block may have no terminator if the next instruction is a jump destination.
-			IRBuilder{it}.CreateBr(nextBlock);
+			IRBuilder{&(*it)}.CreateBr(&(*nextBlockIter));
```

Comment:
I know!   Beautiful, eh?   Clearer in the latest code, but the {} brackets really don't help.